infra/gcp/clusters/projects/k8s-infra-public-pii/provider.tf

```
@@ -0,0 +1,25 @@
+/*
+This file defines:
+- Required provider versions
+- Storage backend details
+*/
+
+terraform {
+
+  backend "gcs" {
+    bucket = "k8s-infra-tf-public-ii"
```

Or maybe I'm misunderstanding the intent here.  I assumed this was about PII, not about the `ii` folks having a dedicated project in public
```suggestion
    bucket = "k8s-infra-tf-public-pii"
```

---

Fixed